index.html

```diff
@@ -989,15 +989,15 @@ <h2>Determining supported touch behavior</h2>
           <h2>Details of <code>touch-action</code> values</h2>
           <dl>
             <dt>auto</dt>
-            <dd>The user agent MAY determine any permitted touch behaviors, such as panning and zooming of the viewport, for touches that begin on the element.</dd>
+            <dd>The user agent MAY determine any permitted touch behaviors related to viewport panning/scrolling and zooming for touches that begin on the element.</dd>
             <dt>none</dt>
-            <dd>Touches that begin on the element MUST NOT trigger default touch behaviors.</dd>
+            <dd>Touches that begin on the element MUST NOT trigger default touch behaviors related to viewport panning/scrolling and zooming for touches that begin on the element.</dd>
             <dt>pan-x<br>pan-left<br>pan-right<br>pan-y<br>pan-up<br>pan-down</dt>
-            <dd>
-              <p>The user agent MAY consider touches that begin on the element only for the purposes of scrolling that starts in any of the directions specified by all of the listed values.  Once scrolling is started, the direction may be reversed by the user even if scrolls that start in the reversed direction are disallowed. In contrast, when scrolling is restricted to starting along a single axis (eg. <code>pan-y</code>), the axis cannot be changed during the scroll.<p>
+            <dd>The user agent MAY consider touches that begin on the element only for the purposes of scrolling that starts in any of the directions specified by all of the listed values.  Once scrolling is started, the direction may be reversed by the user even if scrolls that start in the reversed direction are disallowed. In contrast, when scrolling is restricted to starting along a single axis (eg. <code>pan-y</code>), the axis cannot be changed during the scroll.</dd>
             <dt>manipulation</dt>
-            <dd>The user agent MAY consider touches that begin on the element only for the purposes of scrolling and continuous zooming.  Any additional behaviors supported by <code>auto</code> are out of scope for this specification.</dd>
+            <dd>The user agent MAY consider touches that begin on the element for the purposes of panning/scrolling and <strong>continuous</strong> zooming, but MUST NOT trigger other related touch behaviors such as <strong>non-continuous</strong> zooming in specific steps / to a specific element.</dd>
```

Comment:
Not sure what I am missing here but I can't see what the terms "(non-)continuous zooming" mean.  Do they mean the same idea as in the second Note in Sec9.1?  Or we just need to isolate double-tap zoom as a "non-continuous zoom"?

Would this be easier to follow if we say:
> The user agent MAY allow scrolling/panning in any direction (pan-x or pan-y)...

At least that's [what Chrome does](https://source.chromium.org/chromium/chromium/src/+/master:cc/input/touch_action.h;drc=9287e2cd3a6b75b0f640a1c0b4c198aa5d7c9c1f;l=34).

Reply:
In short yes, manipulation means you're allowed to do everything except double-tap-to-zoom, but we're not allowed to say double-tap-to-zoom as that defines a specific gesture implementation. so this was a way to somehow explain the effect without saying those words.

manipulation also allows single finger double-tap-then-hold to zoom in/out with one finger (like chrome does for maps for instance) and pinch-to-zoom. it really only isolates double-tap-to-zoom, at least that's the idea/my understanding. and here, i'm kind of attempting to define what this double-tap-to-zoom is...a non-continuous step-wise gesture

Reply:
does this make sense, or does it need further rewording?

Reply:
We talked about possibly adding a glossary item for "continuous" zooming because so far this term was never used in normative text.  This is tricky to define using only the allowed terms, but let me try anyway:

> **Continuous zoom**: a zoom effect that takes place while one touch pointer remains active.

If this is okay (_please check carefully!_), perhaps we don't need to define "non-continuous", so can drop "...in specific steps / to a specific element".  Thoughts?